optaplanner-docs/src/modules/ROOT/pages/release-notes/release-notes-8.adoc

```diff
@@ -10,6 +10,14 @@ While the OptaPlanner operator remains experimental, it has now become a part of
 
 If you want to learn more about the operator, follow the https://github.com/kiegroup/optaplanner-quickstarts/tree/development/technology/kubernetes[Kubernetes demo]
 
+==== New `@ShadowVariable` and `@PiggybackShadowVariable` annotations replace the `@CustomShadowVariable`
```

Comment:
Maybe introduce a link to the docs on shadow variables? Just so users don't need to go look it up.